src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowOperationVisitor.cs

```diff
@@ -2757,8 +2757,14 @@ public override TAbstractAnalysisValue VisitConversion(IConversionOperation oper
         {
             var operandValue = Visit(operation.Operand, argument);
 
+            if (!operation.Conversion.Exists)
+                return ValueDomain.UnknownOrMayBeValue;
+
+            if (operation.Conversion.IsImplicit)
```

Comment:
This new logic makes sense for TaintedDataAnalysis, but not so sure about everything else. @mavasani, what do you think?

It might be better to override `VisitConversion()` in `TaintedDataOperationVisitor`.

Reply:
Agreed. Not sure we can fallback to operand value for user defined implicit conversions for all analyses. Can we move the change to `TaintedDataOperationVisitor`?

Reply:
Good point